src/containers/AdminIncomingMessageList/components/IncomingMessageFilter.tsx

```diff
@@ -449,6 +487,7 @@ const IncomingMessageFilter: React.FC<IncomingMessageFilterProps> = (props) => {
           <TextField
             className={css(styles.fullWidth)}
             onChange={onContactNameChanged}
+            defaultValue={contactFilterDefault}
```

Comment:
Had to go back on this too. We use a function to trim the name components, and trying to add a space after the first word would not happen because the spaces would get trimmed. I can make it controlled by setting the `name` as a full value rather than the individual `firstName, lastName, cellNumber`, but that complicates things some more.